meetup/models.py

```diff
@@ -1,29 +1,24 @@
 from django.db import models
```

Comment:
Nitpick: Try to follow the Two Scoops conventions on module imports: Python standard library first, then Django, then third-party dependencies, then your custom code, each block separated by a line of whitespace.